DuckDuckGo/Base.lproj/Bookmarks.storyboard

```diff
@@ -16,11 +17,56 @@
         </array>
     </customFonts>
     <scenes>
+        <!--Favorites View Controller-->
+        <scene sceneID="fa7-3d-Yxy">
+            <objects>
+                <viewController id="pYs-v1-gvr" customClass="FavoritesViewController" customModule="DuckDuckGo" customModuleProvider="target" sceneMemberID="viewController">
+                    <layoutGuides>
+                        <viewControllerLayoutGuide type="top" id="urO-HF-PLt"/>
+                        <viewControllerLayoutGuide type="bottom" id="pjw-Jr-9f0"/>
+                    </layoutGuides>
+                    <view key="view" contentMode="scaleToFill" id="yLW-ha-WmV">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="510"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="22" translatesAutoresizingMaskIntoConstraints="NO" id="FPe-OG-g5a">
+                                <rect key="frame" x="22" y="82" width="331" height="157.5"/>
+                                <subviews>
+                                    <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="BookmarkFavoritesHero" translatesAutoresizingMaskIntoConstraints="NO" id="FRF-MC-5eG">
+                                        <rect key="frame" x="0.0" y="0.0" width="331" height="115.5"/>
+                                        <constraints>
+                                            <constraint firstAttribute="height" constant="115.40000000000001" id="nKK-XC-tDl"/>
+                                        </constraints>
+                                    </imageView>
+                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="No favorites added yet" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="2" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="y8y-oO-xBm">
```

Comment:
I don't think this is using the localized string – I see we do have `UserText.emptyFavorites`, but it isn't used and likely should be here

Reply:
Just updated translations actually.  I've removed this.  Thanks!